axelrod/strategies/darwin.py

```diff
@@ -23,7 +23,14 @@ class Darwin(Player):
     """
 
     name = "Darwin"
-    memory_depth = float('inf')
+    behaviour = {
```

Comment:
This has been addressed by fb079f9 (changing the names - if we're not happy can always change back) and e0d1ed5 (sort out Darwin).